library/junos_ping

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+
+# Copyright (c) 1999-2016, Juniper Networks Inc.
+#               2016, Damien Garros
+#
+# All rights reserved.
+#
+# License: Apache 2.0
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright
+#   notice, this list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright
+#   notice, this list of conditions and the following disclaimer in the
+#   documentation and/or other materials provided with the distribution.
+#
+# * Neither the name of the Juniper Networks nor the
+#   names of its contributors may be used to endorse or promote products
+#   derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY Juniper Networks, Inc. ''AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL Juniper Networks, Inc. BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+DOCUMENTATION = '''
+---
+module: junos_ping
+author: Damien Garros, Juniper Networks
+version_added: "1.3.1"
+short_description: execute ping on junos devices
+description:
+    - execute ping on junos devices
+requirements:
+    - junos-eznc >= 1.2.2
+options:
+  host:
+    description:
+        - Set to {{ inventory_hostname }}
+    required: true
+  user:
+    description:
+        - Login username
+    required: false
+    default: $USER
+  passwd:
+    description:
+        - Login password
+    required: false
+    default: assumes ssh-key active
+  port:
+    description:
+        - TCP port number to use when connecting to the device
+    required: false
+    default: 830
+  dest_ip:
+    description:
+        - Destination ip
+    required: true
+  source_ip:
+    description:
+        - Source IP used to send the ping
+    required: false
+  ttl:
+    description:
+        - Number of device allowed between source and destination
```

Comment:
Suggest changing to:
"Maximum number of IP routers (IP hops) allowed between source and destination"